sources/pmwebapi.go

```diff
@@ -110,7 +110,44 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
-	name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	return name
+}
+
+func fixNaming(name string) string {
+	name = strings.ToLower(name)
+	switch name {
+	case "seconds":
```

Comment:
This is the right idea. Do you have the case statements for cases that require no update because without them you catch their substrings still? If so, I'd say at least remove the newlines that you have placed below cases that require no changes. It's worth taking a look at this link (https://github.com/golang/go/wiki/Switch) for other places you can make optimizations. For example, you could collapse "bytes_byte" and "byte" into a single case statement since they require the same change.

I'm also wondering if this code actually works...you're passing the entire name as the switch case, which I don't think will have an exact match to any of your cases. You're likely going to need to search for the case substrings you've identified within the larger string, I figure.

Reply:
the cases that require no update are the ones I left blank
and youre right I need to find a way to search for the smaller substring in the larger string